doc/nrf/ug_tfm.rst

```diff
@@ -104,7 +102,26 @@ The logs arrive on different COM ports on the host PC.
    * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
      Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
 
-Limitations
-***********
+, at this time, not all SPM functions are available in TF-M.
+
+Migrating from SPM:
```

Comment:
At first I was wondering if this sections should be moved further up, but maybe it is ok to leave it where it is.